reconcile/terraform_resources.py

```diff
@@ -415,8 +417,8 @@ def fetch_current_state(dry_run, namespaces, thread_pool_size,
     return ri, oc_map
 
 
-def init_working_dirs(accounts, thread_pool_size,
-                      oc_map=None, settings=None):
+def init_working_dirs(accounts: list[dict[str, Any]], thread_pool_size: int,
+                      oc_map: Optional[OCMMap] = None, settings: Optional[Mapping[str, Any]] = None) -> tuple[Terrascript, dict[str, str]]:
```

Comment:
Line too long?

Reply:
Let's see if Black catches it. Everything is green on my laptop.

Reply:
It looks like black isn't running against `reconcile/utils`, possibly because it doesn't follow subdirectories? I wonder if @fishi0x01 ran into anything like this when originally adding it? I've fixed the lines manually for now.

Reply:
Nevermind... these are some of the files that are being excluded due to being changed in the last 30 days.

Reply:
You are right - we are in the process of reducing that list :)